src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -1,58 +1,50 @@
 .pf-c-chip-group {
   @include pf-t-light; // force the container follow the light theme
 
-  --pf-c-chip-group--MarginRight: var(--pf-global--spacer--xs);
-  --pf-c-chip-group--MarginBottom: calc(var(--pf-c-chip-group--c-chip--MarginBottom) * -1);
```

Comment:
Removing this negative margin will create a margin underneath chips in a chip group, since the chips have margin.

<img width="369" alt="Screen Shot 2020-04-15 at 5 25 16 PM" src="https://user-images.githubusercontent.com/35148959/79394924-2beeed80-7f3e-11ea-86f7-4e55b5699f64.png">

Reply:
ah good catch

Reply:
Adding the margin bottom of -4px but does this mean that we could run into the error where the component is taller than what is on screen.